Packaging.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <PreReleaseLabel>preview2</PreReleaseLabel>
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
-    <PackageLicenseFile>$(ProjectDir)pkg/dotnet_library_license.txt</PackageLicenseFile>
+    <PackageLicenseFile>$(ProjectDir)LICENSE</PackageLicenseFile>
     <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
```

Comment:
Excellent point. I wasn't aware of the package-specific versions.

We should use the repo-wide 3PN file.

The advantage of the package-specific versions is that they can be accurate for the package. The disadvantage is that this is only the case if the file is kept up to date and we have a perfect understanding of the code that is pulled into a given package (at the point of compilation). This is highly unlikely. As a result, I would suggest that we use the repo-wide 3PN file. It will over-document dependencies, but that is better than under-documenting. Also, if we feel at some point that this is unacceptable, we are free to fix it.